### PR TITLE
Add new OBA classes for COA

### LINKED ISSUE #191: Requesting CPATH related OBA traits
| ID | Label |
|-------------|----------------------------------------|
| NBO:0000351 | body coordination                      |
| NBO:0000342 | limb coordination                      |
| NBO:0000346 | upper limb coordination                |
| NBO:0000345 | lower limb coordination                |
| NBO:0000339 | motor coordination                     |
| NBO:0000347 | gross motor coordination               |
| NBO:0000348 | fine motor coordination                |
| NBO:0000341 | eye-hand coordination                  |
| OBA:XXXXXX1 | whole organism stability               |
| OBA:XXXXXX2 | facial muscle performance              |
| OBA:XXXXXX3 | peripheral nervous system function     |
| OBA:XXXXXX4 | knee joint functionality               |
| OBA:XXXXXX5 | ankle joint functionality              |
| OBA:XXXXXX6 | tendon of biceps brachii functionality |
| OBA:XXXXXX7 | brachioradialis functionality          |

@Daniel-Olson 

Can you provide:

1. Uberon terms (with labels and Ids) for all of these?
2. PATO qualities (with labels and ids) for all of these?

I will deal with politics around NBO.

### PR DESCRIPTION
**Pull request by matentzn**
fixes #191

@ehartley can you check we didn't accidentally add duplicates? EDIT: don't worry lets see if QC complains.

### CODE REVIEW

**Comment by matentzn** [comment]
@rays22 the main issue here is to 

- [ ] Add to [this](https://github.com/obophenotype/bio-attribute-ontology/blob/master/src/ontology/oba-odk.yaml#L23) the full IRIs (not the prefix, including the numbers!) of limb coordination and motor coordination.
- [ ] Refresh merged import
- [ ] Manually add a subclass axioms in `src/ontology/oba-edit.obo` for those two that cannot automatically be classified.

File: src/patterns/data/default/entity_attribute.tsv [hunk 1]
```diff
@@ -4156,3 +4156,15 @@ OBA:2060170		UBERON:0002101	limb	PATO:0000140	position
 OBA:2060171		UBERON:0000966	retina	PATO:0000141	structure
 OBA:2060172		UBERON:0005056	external female genitalia	PATO:0000070	amount
 OBA:2060173		UBERON:0000970	eye	PATO:0000070	amount
+OBA:2060179	body coordination	UBERON:0000468	multicellular organism	PATO:0000188	coordination
+OBA:2060180	limb coordination	UBERON:0002101	limb	PATO:0000188	coordination
+OBA:2060181	upper limb coordination	UBERON:0002102	forelimb	PATO:0000188	coordination
+OBA:2060182	lower limb coordination	UBERON:0002103	hindlimb	PATO:0000188	coordination
+OBA:2060183	motor coordination	UBERON:0025525	motor system	PATO:0000188	coordination
```
Comment (rays22) [inline src/patterns/data/default/entity_attribute.tsv:4163]: There is a similar [NBO:0000591 motor coordination phenotype](https://www.ebi.ac.uk/ols/ontologies/NBO/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FNBO_0000591)

**Comment by rays22** [comment]
> * When QC passes, review the remaining terms.

QC will not pass until https://github.com/obophenotype/bio-attribute-ontology/pull/211 has been merged.

File: src/patterns/data/default/entity_attribute.tsv (hunk 1 above)
Comment (matentzn) [reply src/patterns/data/default/entity_attribute.tsv:4163]: We are about to move all of these to OBA, so lets keep a mental note of that!

**Review by rays22 — CHANGES_REQUESTED** [review]
@matentzn , is it OK if I remove most of the 'coordination' terms? I think we need to think about how to best patternise them.

**Review by rays22 — APPROVED** [review]

**Comment by rays22** [comment]
@matentzn , I can not see the QC check results on this page as usual, although the [ontology_qc action seems to have run successfully.](https://github.com/obophenotype/bio-attribute-ontology/actions/runs/4138961328) 
Is it OK to merge this PR now?

**Comment by matentzn** [comment]
Ok!

**Pull request merged**